openeo/extra/job_management.py

```diff
@@ -886,3 +937,124 @@ def create_job_db(path: Union[str, Path], df: pd.DataFrame, *, on_exists: str =
     else:
         raise NotImplementedError(f"Initialization of {type(job_db)} is not supported.")
     return job_db
+
+
+class UDPJobFactory:
```

Comment:
TODO: find better class name?

Reply:
bit of brainstorming

- use `Creator` instead of `Factory`, to sound less technical?
- avoid over-specificity of "UDP"  and use more general, but considerably longer "ParameterizedProcess" ?
- the class could even be decoupled further from the job manager aspect, to just wrap a process definition and have methods to create a cube, given parameter values (leaving the job creation to the user). From that standpoint, the essential nature of the class is to concertize a parametrized process definition in some way (create a DataCube, create a batch job, ...)

Reply:
let's leave this to a todo note for now